src/plugins/foxPage/hooks/getGovernanceData.ts

```diff
@@ -13,7 +13,7 @@ type BoardroomGovernanceData = Array<{
 
 const BOARDROOM_API_BASE_URL = getConfig().REACT_APP_BOARDROOM_API_BASE_URL
 
-const parseGovernanceData = (governanceData: BoardroomGovernanceData) => {
+export const parseGovernanceData = (governanceData: BoardroomGovernanceData) => {
```

Comment:
Can we type the return type for safety?

Reply:
Will apply it from https://github.com/shapeshift/web/pull/2023/files#diff-19a3f5e2593679b4a282f7c7cf88a157bbf3ab2d97819f695f38e06aa2637a4fR26

Reply:
Added in 03be9dcbb0216be02c732d5cbef2058a0dff9517